### ISSUE TITLE
Unable to add custom image

### ISSUE THREAD

**Issue by stewe12** [coolzhao/Geo-SAM#1]
Everytime I try to use different custom image for segmentation it says: Point is located outside of the image boundary. I tried different rasters, different projections, WMS, XYZ tiles... Is there a way how to change boundary where I want to do the segmentation?

I am on Windows 11 64bit, QGis 3.30, tried plugins version 1.0.2 and dev 1.0.3.

It is probably user error, but I am not able to find how to fix that.

Anyhow, thank you for the work you are doing! The sample looks really promising!

Stepan

**Comment by coolzhao** [comment]
Hi Stepan, thank you for your interest. Sorry to let you know that our tool currently only supports preprocessed images (whose features have been generated in advance using a separate program). We are now building a tool for encoding images inside QGIS, which will soon be available. I will let you know when it's ready, so stay tuned.

**Comment by stewe12** [comment]
Oh I see. So it was user error as I expected :-) Is there a way how I can preprocess them myself?

I am asking, because I tried different implementation of SAM for GIS use and your really looks the best for my use case. I need semi guided ortho classification, mainly on middle eastern maps.

For example I tried using this project: https://github.com/opengeos/segment-geospatial But on my maps it had mixed results. European ortho was working good, but middle easter is much harder to recognize features and it struggled hard. A that is where I expect your implementation be really superior with human inputs.
